### PR TITLE
Feature/erez edits

### PR DESCRIPTION
**Pull request by ErezBinyamin**
I made your bash scripts more reliable, added some pretty colors, and put in a warning for users before you go ahead and sudo replace their entire /etc/ directory lol

### CODE REVIEW

**Comment by ErezBinyamin** [comment]
uninstall.sh needs work too... I think when you "install.sh" you should "cp" all the old files to the "Original" directory(before you overwrite them), then when you "uninstall.sh" you just replace them.

I think this is what your intention was, but that's not currently how "uninstall.sh" works.

Let me know if that sounds like something I should do, and I can whip it together real quick

File: Scripts/addNetwork.sh [hunk 1]
```diff
@@ -1,16 +1,47 @@
...
+# Usage:
+#	./addNetwork.sh <SSID> <Password>
 
 DIR=/etc/wpa_supplicant/wpa_supplicant.conf
 SSID=$1
 PASSWORD=$2
 
+# Colors
+ERROR=$(tput setaf 1)
+SUCESS=$(tput setaf 2)
```
Comment (anthonyguella) [inline Scripts/addNetwork.sh:13]: Typo

File: Scripts/addNetwork.sh [hunk 2]
```diff
@@ -1,16 +1,47 @@
 #!/bin/bash
+# Configures a network with a given SSID and optional Password
+# by edditing the configuration file specified by "DIR"
```
Comment (anthonyguella) [inline Scripts/addNetwork.sh:3]: Typo

File: install.sh [hunk 3]
```diff
@@ -1,32 +1,65 @@
...
+# OVERWRITES /etc/ config files
 
-# Install Dependencies
-sudo apt-get update
-sudo apt-get upgrade
-sudo apt-get install hostapd
-sudo apt-get install dnsmasq
+# Colors
+ERROR=$(tput setaf 1)
+SUCESS=$(tput setaf 2)
```
Comment (anthonyguella) [inline install.sh:7]: Typo

File: install.sh [hunk 4]
```diff
@@ -1,32 +1,65 @@
...
-# Disable Services
-sudo systemctl disable hostapd
-sudo systemctl disable dnsmasq
+# User warning
+printf "\n\n${ERROR}WARNING:${NORMAL} This script will overwrite your system configuration files:\n"
+printf "\n\t/etc/hostapd/hostapd.conf"
+printf "\n\t/etc/default/hostapd"
+printf "\n\t/etc/dnsmasq.conf"
+printf "\n\t/etc/dhcpcd.conf"
+printf "\n\t/usr/bin/hotspotWebserver.py"
```
Comment (anthonyguella) [inline install.sh:17]: Line 17-20 are my scripts. They aren't being overwritten. We should leave the other warnings though

File: install.sh [hunk 5]
```diff
@@ -1,32 +1,65 @@
...
+printf "\n\t/usr/bin/addNetwork.sh"
+printf "\n\t/etc/systemd/system/autohotspot.service"
+printf "\n\t/usr/bin/autohotspotN"
 
-# Hostapd Configuration
-sudo cp ./Config/hostapd.conf /etc/hostapd/hostapd.conf
-sudo cp ./Config/hostapd /etc/default/hostapd
+printf "\n\nDo you wish to continue? [Y/N]: "
+read CHOICE
+if [[ ${CHOICE,,} == "y" ]]
```
Comment (anthonyguella) [inline install.sh:24]: Does this also work with capital y? ("Y")

File: install.sh [hunk 6]
```diff
@@ -1,32 +1,65 @@
...
+	sudo cp ./Scripts/addNetwork.sh /usr/bin/addNetwork.sh
+
+	# APModeService Setup
+	sudo cp ./Scripts/autohotspot.service /etc/systemd/system/autohotspot.service
+	sudo cp ./Scripts/autohotspotN /usr/bin/autohotspotN
+
+	# Enable auto hotspot service
+	sudo systemctl enable autohotspot.service
+else
+	printf "\nBye :)\n"
```
Comment (anthonyguella) [inline install.sh:64]: Change to "Installation cancelled."

**Review by anthonyguella — CHANGES_REQUESTED** [review]
Check comments

File: Scripts/addNetwork.sh [hunk 7]
```diff
@@ -1,16 +1,47 @@
...
     echo "  psk=\""$PASSWORD"\"" >> $DIR
 else
     echo "  key_mgmt=NONE" >> $DIR
 fi
-echo "}" >> $DIR
\ No newline at end of file
+echo "}" >> $DIR
+
+
+printf "\nNetwork added successfully!\n\n"
```
Comment (anthonyguella) [inline Scripts/addNetwork.sh:45]: Was this supposed to be in the success color?

File: install.sh (hunk 5 above)
Comment (ErezBinyamin) [reply install.sh:24]: Yup the “,,” makes it lowercase

File: Scripts/addNetwork.sh (hunk 7 above)
Comment (ErezBinyamin) [reply Scripts/addNetwork.sh:45]: Nahhh. I usually only use that color for conditional success.

**Pull request closed without merging**